service/controller.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/dell/csi-isilon/v2/common/utils"
 	isi "github.com/dell/goisilon"
 	isiApi "github.com/dell/goisilon/api"
+	v1 "github.com/dell/goisilon/api/v1"
```

Comment:
It's already named v1, why keep the alias?

Reply:
Tried removing it, its getting added automatically.

Reply:
ok